File: covertit/app/src/main/java/com/example/covertit/MainActivity.java
package com.example.covertit;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    private Button button;
    private TextView textview;
    private EditText editText;

    public MainActivity() {
    }

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        button=findViewById(R.id.button2);
        textview=findViewById(R.id.textView3);
        editText=findViewById(R.id.editTextTextPersonName);
        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Toast.makeText(MainActivity.this, "hi click listener worked!", Toast.LENGTH_SHORT).show();
                String s = editText.getText().toString();
                int km=Integer.parseInt(s);
                double miles=0.621371 * km;
                textview.setText(miles + " miles ");





            }
        });
    }
}

File: meme app(Kotlin)/app/src/main/java/com/example/memeapp/MainActivity.kt
package com.example.memeapp

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {

    var currentImageUrl: String? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun nextMeme(view: View) {


    }
}